contracts/vaults/RibbonThetaVault.sol

```diff
@@ -399,56 +448,17 @@ contract RibbonThetaVault is DSMath, OptionsVaultStorage {
         require(amount > 0, "!amount");
         require(!depositReceipt.processed, "Processed");
         require(depositReceipt.round == currentRound, "Invalid round");
-        uint128 receiptAmount = depositReceipt.amount;
+        uint104 receiptAmount = depositReceipt.amount;
         require(receiptAmount >= amount, "Exceed withdraw amount");
 
-        // Subtraction underflow checks already ensure it is smaller than uint128
-        depositReceipt.amount = uint128(uint256(receiptAmount).sub(amount));
+        // Subtraction underflow checks already ensure it is smaller than uint104
+        depositReceipt.amount = uint104(uint256(receiptAmount).sub(amount));
 
         emit InstantWithdraw(msg.sender, amount, currentRound);
 
         transferAsset(msg.sender, amount);
     }
 
-    /**
-     * @notice Lock's users shares for future withdraw and ensures that the new short excludes the scheduled amount.
-     * @param shares is the number of shares to be withdrawn in the future.
-     */
-    function withdrawLater(uint256 shares) external nonReentrant {
```

Comment:
why exactly we removing this stuff? are you just going to put it back in with the subsequent withdraw later pr?

Reply:
I was running into bytecode size problems so removing them was a hack haha. Plan to add them back